source/Background/index.ts

```diff
@@ -3,7 +3,7 @@
  *
  * ZAP is an HTTP/HTTPS proxy for assessing web application security.
  *
- * Copyright 2022 The ZAP Development Team
+ * Copyright 2023 The ZAP Development Team
```

Comment:
You dont have to go back an update old Copyright years. Doesnt hurt in the case, so just an FYI. We only need to put the current year in new files.